GOVERNACE.md

```diff
@@ -1,45 +1,75 @@
+### Organizer Committee
 
-## Developer Group Structure
+The Organizer Committee consists of organizers in charge of event or activity operations. They are in charge of the execution of PMC’s strategies and decisions. Specifically, the members are the leaders of user groups or community event organizers across regions.
+
+### Developer Group
+
+As the cornerstone of community development, the TiKV Developer Group consists of these roles: Maintainer, Committer, Reviewer, Active Contributor, and Contributor. Each role takes corresponding responsibilities in the community. Collectively they play an important role in the robust development of TiDB.
+
+The Developer Group operates TiKV projects in two function forms: Special Interest Group (SIG) and Working Group (WG). The diagram below illustrates the definition of the SIG and WG, the internal roles of the group, and the role promotion path.
 
 ![developer group](/media/governace/developer_group.png)
 
-Community groups include Special Interest Group (SIG) and Working Group (WG). The second part of the organizational structure illustrates the definition of the SIG and WG, the internal roles of the group, and the role promotion machanism.
+#### SIGs
+
+The TiKV project is organized primarily into Special Interest Groups, or SIGs. Each SIG is comprised of members from multiple companies and organizations, with a common purpose of advancing the project with respect to a specific topic, such as Coprocessor or Documentation. Our goal is to enable a distributed decision structure and code ownership, as well as providing focused forums for getting work done, making decisions, and onboarding new contributors. Every identifiable subpart of the project (e.g., github org, repository, subdirectory, API, test, issue, PR) is intended to be owned by some SIG.
+
+Currently SIG membership requires invitation - potential qualified members are Active Contributors in a particular module. It is our intention to properly guide or mentor community talents, and help them advance to the higher-level Reviewer, Committer and Maintainer within an SIG.
+
+Each SIG must have a charter that specifies its scope (topics, code repos and directories), responsibilities, areas of authority, how members and roles of authority/leadership are selected/granted, how decisions are made, and how conflicts are resolved. See the SIG charter process for details on how charters are managed. SIGs should be relatively free to customize or change how they operate, within some broad guidelines and constraints imposed by cross-SIG processes.
+
+See [sig governance](../sig-governace/sig-governace.md) for more details about current SIG operating mechanics, such as mailing lists, meeting times, etc.
```

Comment:
`../sig-governace/sig-governace.md` -> `/sig-governace/sig-governace.md`, absolute path is more appreciate